tests/conftest.py

```diff
@@ -1,10 +1,13 @@
 import os
 
 import pytest
+from dotenv import load_dotenv
 
 from unstract.llmwhisperer.client import LLMWhispererClient
 from unstract.llmwhisperer.client_v2 import LLMWhispererClientV2
 
+load_dotenv()
```

Comment:
@nagesh-zip this is not required since we have [pytest-dotenv](https://github.com/Zipstack/llm-whisperer-python-client/blob/1fc51fe202bc304b9029f3253e7550707a65346c/pyproject.toml#L35) added. It will [pick up and load dotenv files automatically](https://pypi.org/project/pytest-dotenv/).

Moreover, if we import a new dependency `from dotenv import load_dotenv` we need to add this into `pyproject.toml` to not break tests. Please remove this and try to run the tests to confirm (you might have to place `.env` in `/tests` for this)